### DISCUSSION TITLE
Can someone tell me how to get values from an object with condition

### DISCUSSION THREAD

**Discussion by Makskknd** [pimcore/pimcore#9962] [General]
Good afternoon, can someone tell me how to get values from an object with conditions, at the moment I know how to get object values by id
$news = MyCategory::getById(90401);

how do I select values from an object with certain values ?
$params = AdmtadCategory::getById(['AdmitadParentCategoryID'=>'90401']);

**Comment by Humb3l** [comment]
Hey, 
here is everything u need:

https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Working_with_PHP_API.html
